ygen/genstate.go

```diff
@@ -199,7 +206,7 @@ func (s *genState) buildDirectoryDefinitions(entries map[string]*yang.Entry, com
 
 			// Find the elements that should be rooted on this particular entity.
 			var fieldErr []error
-			elem.fields, fieldErr = findAllChildren(e, compressPaths)
+			elem.fields, fieldErr = findAllChildren(e, compressPaths, excludeConfigFalse)
```

Comment:
(optional) I am wondering if "config false" is the best possible terminology we can choose. Is it a well known term in this codebase? generally it is better to describe a thing by saying what it is rather than what it isn't. I feel that excludeState or excludeReadOnly would be more obvious.

Up to you, feels like this would be a large change to ask for.

Reply:
RFC6020 Section 4.2.3 is where this is defined: https://tools.ietf.org/html/rfc6020#section-4.2.3  -- I've tried to be as consistent as possible with the RFC, but I agree with this comment. Refactored throughout.